locations/Bahraich/inputs/impact/ghg_inputs.yaml

```diff
@@ -0,0 +1,43 @@
+################################################################################
+# ghg_inputs.yaml - GHG input information.                                     #
+#                                                                              #
+# Author: Phil Sandwell, Ben Winchester                                        #
+# Copyright: Phil Sandwell & Ben Winchester, 2021                              #
+# Date created: 14/07/2021                                                     #
+# License: Open source                                                         #
+################################################################################
+
+general_o&m_ghgs: 200 # [kgCO2 p.a.]
+misc_ghgs: 0 # [kgCO2/kW]
+bos:
+  ghgs: 200 # [kgCO2/kW]
+  ghg_decrease: 2 # [% p.a.]
+diesel_fuel:
+  ghgs: 2 # [kgCO2/litre]
+  o&m_ghgs: 10 # [kgCO2/kW p.a.]
+dieel_generator:
+  ghgs: 2000 # [kgCO2/kW]
+  ghg_decrease: 0 # [% p.a.]
+  installation_ghgs: 50 # [kgCO2/kW]
+  installation_ghg_decrease: 0 # [% p.a.]
+grid:
+  extension_ghgs: 290000 # [kgCO2/km]
+  infrastructure_GHGs: 1200000 # [kgCO2]
+  initial_ghgs: 0.8 # [kgCO2/kWh]
+  final_ghgs: 0.4 # [kgCO2/kWh]
+households:
+  connection_ghgs: 10 # [kgCO2/household]
+  kerosene_ghgs: 0.055 # [kgCO2/hour]

```

Comment:
As in `finance_inputs.yaml` I think there should be a separate `kerosene` section